javascript/javascript3/week3/randomCircle.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Random circle</title>
+</head>
+<body>
+    <canvas id="canvas" height=900px, width=800px ></canvas>
+
+    <script>
+setInterval(function draw() {
+  const creatCanvas = document.getElementById("canvas");
+  const context = creatCanvas.getContext("2d");
+  context.beginPath();
+  context.arc(
+    Math.floor(Math.random() * 500),
+    Math.floor(Math.random() * 600),
+    Math.floor(Math.random() * 400),
```

Comment:
You've used `Math.floor(Math.random() * x)` already 3 times here. It would make sense to write a function to get random number.